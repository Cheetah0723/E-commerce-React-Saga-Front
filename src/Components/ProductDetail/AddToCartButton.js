import React from 'react';
import Button from '@material-ui/core/Button';

const style = {
    paddingTop: 10, height: "4rem",
    marginTop: 10, borderRadius: "25px", color: "white", backgroundColor: "#007db5"
};
const AddToCartButton = ({ onAddToCart }) => {
    return <Button variant="outlined"
        onClick={onAddToCart}
        style={style}><b>Add to Cart</b></Button>
}

export default AddToCartButton;