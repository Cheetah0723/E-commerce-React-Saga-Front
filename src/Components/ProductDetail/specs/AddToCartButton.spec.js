import React from 'react';
import AddToCartButton from "../AddToCartButton"
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const test = (arg) => { console.log(arg) }
    const component = renderer.create(<AddToCartButton onAddToCart={test} />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})


//TODO failed