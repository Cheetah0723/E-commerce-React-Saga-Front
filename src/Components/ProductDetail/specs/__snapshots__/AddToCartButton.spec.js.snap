// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<button
  className="MuiButtonBase-root MuiButton-root MuiButton-outlined"
  disabled={false}
  onBlur={[Function]}
  onClick={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  type="button"
>
  <span
    className="MuiButton-label"
  >
    <b>
      Add to Cart
    </b>
  </span>
</button>
`;
